auth0/resource_auth0_organization_test.go

```diff
@@ -154,3 +155,90 @@ resource auth0_organization acme {
 	}
 }
 `
+
+func TestAccOrganizationAssignUsers(t *testing.T) {
+	httpRecorder := configureHTTPRecorder(t)
+
+	testName := strings.ToLower(t.Name() + fmt.Sprintf("%d", rand.Intn(99))) // TODO: REMOVE! This is only here to get around annoying 409s
```

Comment:
As the TODO suggests, this was just to make the TDD process more smooth. Otherwise several 409s would occur.